CHANGELOG.md

```diff
@@ -1,3 +1,44 @@
+CHANGES IN VERSION 1.13.0 (FROM 1.12.1)
+===================================
+
+Appium 1.13.0 is a minor release
+
+#### General
+* Appium 1.13 will be the last minor version to use [UiAutomator1](https://github.com/appium/appium-android-driver) as the default Android automation. As of Appium 1.14, the default Android driver will be [UiAutomator2](https://github.com/appium/appium-uiautomator2-driver). If you don't set the `automationName` for an Android session a big [warning](https://github.com/appium/appium/pull/12481) will be displayed notifying you of the change. This will be a breaking change, and it's recommended if you wish to keep the same behavior, add the capability `automationName=UiAutomator1` to your scripts
+* Appium 1.13 will be the last minor version to support Node v8. As of Appium 1.14 the supported Node versions will be v10 and v12.
+
```

Comment:
- what about bumping up https://github.com/appium/appium-base-driver/commits/master , too?
- we can address _Added `defaultImageTemplateScale` to arrange image comparison logic [#307](https://github.com/appium/appium-base-driver/pull/307). Read [doc](docs/en/advanced-concepts/image-elements.md) for more details._ in https://github.com/appium/appium/pull/12382/files . It is already included in this version